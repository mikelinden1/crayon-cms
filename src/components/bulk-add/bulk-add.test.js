import BulkAdd from './bulk-add';
import configSample from 'config-sample/widget-config';

const props = {
    columns: [],
    open: true,
    adding: false,
    config: configSample,
    actions: {
        setColumns: () => {},
        addColumn: () => {},
        closeDialog: () => {},
        openDialog: () => {},
        throwBulkError: () => {},
        save: () => {}
    }
};

it('BulkAdd matches snapshot', () => {
    const wrapper = mount(<BulkAdd {...props} />);
    expect(wrapper).toMatchSnapshot();
});