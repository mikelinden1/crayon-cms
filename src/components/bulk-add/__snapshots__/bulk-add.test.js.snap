// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BulkAdd matches snapshot 1`] = `
<div>
  <button
    className="link-btn btn btn-secondary"
    onClick={[Function]}
    type="button"
  >
    Bulk Import
  </button>
  <Portal
    containerInfo={
      <div
        tabindex="-1"
        style="position: relative; z-index: 1050;"
      >
        <div
          class=""
        >
          <div
            class="modal fade"
            style="display: block;"
            role="dialog"
            tabindex="-1"
          >
            <div
              class="modal-dialog bulk-add"
              role="document"
            >
              <div
                class="modal-content"
              >
                <form>
                  <div
                    class="modal-header"
                  >
                    <h5
                      class="modal-title"
                    >
                      Bulk Import 
                      Widgets
                    </h5>
                    <button
                      type="button"
                      class="close"
                      aria-label="Close"
                    >
                      <span
                        aria-hidden="true"
                      >
                        ×
                      </span>
                    </button>
                  </div>
                  <div
                    class="modal-body"
                  >
                    <h3>
                      CSV Columns
                    </h3>
                    <div
                      class="react-tagsinput"
                    >
                      <span>
                        <input
                          type="text"
                          class="react-tagsinput-input"
                          placeholder="Add a tag"
                          value=""
                        />
                      </span>
                    </div>
                    <div
                      style="margin: 10px 0px;"
                    >
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        name
                      </button>
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        slug
                      </button>
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        type
                      </button>
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        radio
                      </button>
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        photo
                      </button>
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        start_date
                      </button>
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        checkbox
                      </button>
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        example
                      </button>
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        tags
                      </button>
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        copy
                      </button>
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        country
                      </button>
                      <button
                        type="button"
                        class="btn-sm btn btn-secondary"
                      >
                        canada
                      </button>
                    </div>
                    <p>
                      <small>
                        <em>
                          Click on column names to add them
                        </em>
                      </small>
                    </p>
                    <h3>
                      CSV File
                    </h3>
                    <input
                      type="file"
                      id="csvFile"
                    />
                  </div>
                  <div
                    class="modal-footer"
                  >
                    <button
                      type="submit"
                      class="btn btn-primary"
                    >
                      Add
                    </button>
                    <button
                      type="button"
                      class="btn btn-secondary"
                    >
                      Cancel
                    </button>
                  </div>
                </form>
              </div>
            </div>
          </div>
          <div
            class="modal-backdrop fade"
          />
        </div>
      </div>
    }
  >
    <div
      className=""
    >
      <div
        className="modal fade"
        onClick={[Function]}
        onKeyUp={[Function]}
        role="dialog"
        style={
          Object {
            "display": "block",
          }
        }
        tabIndex="-1"
      >
        <div
          className="modal-dialog bulk-add"
          role="document"
        >
          <div
            className="modal-content"
          >
            <form
              onSubmit={[Function]}
            >
              <div
                className="modal-header"
              >
                <h5
                  className="modal-title"
                >
                  Bulk Import 
                  Widgets
                </h5>
                <button
                  aria-label="Close"
                  className="close"
                  onClick={[Function]}
                  type="button"
                >
                  <span
                    aria-hidden="true"
                  >
                    ×
                  </span>
                </button>
              </div>
              <div
                className="modal-body"
              >
                <h3>
                  CSV Columns
                </h3>
                <div
                  className="react-tagsinput"
                  onClick={[Function]}
                >
                  <span>
                    <input
                      className="react-tagsinput-input"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onPaste={[Function]}
                      placeholder="Add a tag"
                      type="text"
                      value=""
                    />
                  </span>
                </div>
                <div
                  style={
                    Object {
                      "margin": "10px 0",
                    }
                  }
                >
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    name
                  </button>
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    slug
                  </button>
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    type
                  </button>
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    radio
                  </button>
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    photo
                  </button>
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    start_date
                  </button>
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    checkbox
                  </button>
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    example
                  </button>
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    tags
                  </button>
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    copy
                  </button>
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    country
                  </button>
                  <button
                    className="btn-sm btn btn-secondary"
                    disabled={false}
                    onClick={[Function]}
                    type="button"
                  >
                    canada
                  </button>
                </div>
                <p>
                  <small>
                    <em>
                      Click on column names to add them
                    </em>
                  </small>
                </p>
                <h3>
                  CSV File
                </h3>
                <input
                  disabled={false}
                  id="csvFile"
                  onChange={[Function]}
                  type="file"
                />
              </div>
              <div
                className="modal-footer"
              >
                <button
                  className="btn btn-primary"
                  onClick={[Function]}
                  type="submit"
                >
                  Add
                </button>
                 
                 
                <button
                  className="btn btn-secondary"
                  disabled={false}
                  onClick={[Function]}
                  type="button"
                >
                  Cancel
                </button>
              </div>
            </form>
          </div>
        </div>
      </div>
      <div
        className="modal-backdrop fade"
      />
    </div>
  </Portal>
</div>
`;
